packages/components/src/components/drawer/drawer.ts

```diff
@@ -89,32 +78,24 @@ export default class SdDrawer extends SolidElement {
   @property({ type: Boolean, reflect: true }) open = false;
 
   /**
-   * The drawer's label as displayed in the header. You should always include a relevant label even when using
-   * `no-header`, as it is required for proper accessibility. If you need to display HTML, use the `label` slot instead.
+   * You should always include a relevant label even when using
+   * `no-header`, as it is required for proper accessibility.
```

Comment:
is this our standard text element for components where we give away control on a11y? IMO we should point it out more loud (bold, stars, emojis etc) to make clear users destroy one of the most powerful aspects in the design system if they do...